Dockerfile

```diff
@@ -1,18 +1,18 @@
 FROM ghcr.io/linuxserver/baseimage-kasmvnc:alpine317
 
-RUN apk update
-RUN apk add --no-cache android-tools \
-         openjdk17 
+# Install necessary tools and dependencies
+RUN apk update && apk add --no-cache \
+    android-tools \
+    openjdk17 \
+    wget
 
 RUN wget https://redirector.gvt1.com/edgedl/android/studio/ide-zips/2024.1.1.11/android-studio-2024.1.1.11-linux.tar.gz -O /tmp/android-studio.tar.gz \
     && mkdir -p /opt/android-studio \
     && tar -xvzf /tmp/android-studio.tar.gz -C /opt/android-studio --strip-components=1 \
     && rm /tmp/android-studio.tar.gz
 
-RUN mkdir -p /defaults
-# Set up autostart for Android Studio
-RUN echo "/opt/android-studio/bin/studio.sh" > /defaults/autostart
-RUN mkdir -p /config
+RUN mkdir -p /defaults /config \
+    && echo "/opt/android-studio/bin/studio.sh" > /defaults/autostart
```

Comment:
Seems like a reasonable cleanup. I was thinking longer term it probably makes sense to move all the generated files in this dockerfile to be checked in and moved where appropriate. Fine for now.